tests/tests_suite_test.go

```diff
@@ -50,7 +51,12 @@ var _ = BeforeSuite(func() {
 })
 
 var _ = AfterSuite(func() {
-	tests.AfterTestSuitCleanup()
+	// We have to create the artifacts directory before running any cleanup code
+	// because in case of a failure in one of the cleanup steps, we still want to
+	// be able to store artifacts.
+	artifactsDir, err := tests.CreateDirectory(os.Getenv("ARTIFACTS"))
```

Comment:
Artifacts directory created before starting the test suite, if we want to do it from within the test suite 
we should remove this line:
https://github.com/kubevirt/kubevirt/blob/master/automation/test.sh#L286